proto/src/determined/master/v1/master.proto

```diff
@@ -26,3 +26,17 @@ message ResourceAllocationRawEntry {
   // The number of slots used by the allocation.
   int32 slots = 8;
 }
+
+// One instance of slots in the cluster being allocated to a task during a day
+// (aggregated).
+message ResourceAllocationAggregatedEntry {
+  // The type of aggregation performed for this entry ("total").
+  string aggregation_type = 1;
```

Comment:
should we make aggregation type an enum?

Reply:
Yeah I sort of agree, but I also think it _is_ an enum.. it really is grpc gateway that translates them in such an overly-verbose way. and the verbosity is the only really downside.

Reply:
Okay, I made it an enum.